### DISCUSSION TITLE
sshj Version 0.36.0 not on maven central

### DISCUSSION THREAD

**Discussion by WargulWB** [hierynomus/sshj#891] [General]
Hi,

just a quick question: Why is the current version 0.36.0 not on maven central?

See: https://mvnrepository.com/artifact/com.hierynomus/sshj
Or: https://central.sonatype.com/artifact/com.hierynomus/sshj

The README.adoc is somewhat confusing at the moment because of this.

**Comment by hierynomus** [comment]
It's there now!
